build/Dockerfile.ci

```diff
@@ -107,7 +105,6 @@ RUN make build-daemon
 # /payload/
 #├── azure-cloud-node-manager.exe
 #├── cni/
-#│   ├── flannel.exe
```

Comment:
there is another instance in 

https://github.com/openshift/windows-machine-config-operator/blob/ea36c7b1a691bd5273aace390fb58e4f2e0e3c00/build/Dockerfile#L101